_maps/RandomRuins/SpaceRuins/caravanambush.dmm

```diff
@@ -398,764 +650,3542 @@
-/area/ruin/unpowered)
-"dq" = (
-/mob/living/simple_animal/hostile/syndicate/ranged/space{
-	environment_smash = 0;
-	name = "Syndicate Salvage Worker"
+/area/shuttle/caravan/pirate)
+"eR" = (
+/obj/machinery/atmospherics/components/unary/vent_pump/on,
+/turf/open/floor/plasteel/vault{
+	dir = 5
 	},
-/turf/open/floor/mineral/titanium/blue/airless,
-/area/ruin/unpowered)
-"dr" = (
-/obj/effect/decal/cleanable/dirt,
-/obj/item/clipboard,
-/turf/open/floor/mineral/titanium/blue/airless,
-/area/ruin/unpowered)
-"ds" = (
-/obj/structure/shuttle/engine/heater,
-/turf/closed/wall/mineral/plastitanium,
-/area/ruin/powered)
-"dt" = (
-/obj/structure/closet/crate{
-	icon_state = "crateopen";
-	opened = 1
+/area/shuttle/caravan/pirate)
+"eS" = (
+/turf/open/floor/plasteel/vault{
+	dir = 5
 	},
-/obj/item/stack/sheet/mineral/diamond{
-	amount = 5
+/area/shuttle/caravan/pirate)
+"eT" = (
+/obj/structure/chair/office/dark{
+	dir = 4
 	},
-/obj/item/stack/sheet/mineral/gold{
-	amount = 25
+/obj/machinery/turretid{
+	icon_state = "control_kill";
+	lethal = 1;
+	locked = 0;
+	pixel_y = -30;
+	req_access = null
 	},
-/turf/open/floor/mineral/plastitanium,
-/area/ruin/powered)
-"du" = (
-/obj/structure/chair{
+/mob/living/simple_animal/hostile/pirate/ranged{
+	environment_smash = 0
+	},
+/turf/open/floor/plasteel/vault{
+	dir = 5
+	},
+/area/shuttle/caravan/pirate)
+"eU" = (
+/obj/machinery/computer/shuttle/caravan/pirate{
+	dir = 8
+	},
+/turf/open/floor/plasteel/darkred/side{
+	dir = 8
+	},
+/area/shuttle/caravan/pirate)
+"eV" = (
+/obj/machinery/power/smes{
+	charge = 5e+006
+	},
+/obj/structure/cable/yellow{
+	icon_state = "0-4"
+	},
+/turf/open/floor/plating,
+/area/shuttle/caravan/pirate)
+"eW" = (
+/obj/structure/cable/yellow{
+	icon_state = "1-8"
+	},
+/obj/machinery/atmospherics/pipe/simple/supply/hidden,
+/turf/open/floor/plating,
+/area/shuttle/caravan/pirate)
+"eX" = (
+/obj/structure/table,
+/obj/item/storage/box/lethalshot,
+/turf/open/floor/plasteel/vault{
+	dir = 5
+	},
+/area/shuttle/caravan/pirate)
+"eY" = (
+/obj/structure/table,
+/obj/item/grenade/smokebomb{
+	pixel_x = -4
+	},
+/obj/item/grenade/smokebomb{
+	pixel_x = 2
+	},
+/obj/machinery/airalarm/all_access{
+	pixel_y = 24
+	},
+/turf/open/floor/plasteel/vault{
+	dir = 5
+	},
+/area/shuttle/caravan/pirate)
+"eZ" = (
+/obj/structure/tank_dispenser/oxygen,
+/obj/machinery/firealarm{
+	dir = 2;
+	pixel_y = 24
+	},
+/turf/open/floor/plasteel/vault{
+	dir = 5
+	},
+/area/shuttle/caravan/pirate)
+"fa" = (
+/obj/structure/table,
+/obj/structure/window/reinforced{
+	dir = 8
+	},
+/obj/machinery/door/window/southleft{
+	name = "Weapon Storage"
+	},
+/obj/item/gun/ballistic/shotgun/automatic/combat,
+/turf/open/floor/plasteel/vault{
+	dir = 5
+	},
+/area/shuttle/caravan/pirate)
+"fb" = (
+/obj/structure/table,
+/obj/item/gun/energy/laser{
+	pixel_x = -3;
+	pixel_y = 3
+	},
+/obj/item/gun/energy/laser,
+/obj/machinery/door/window/southleft{
+	base_state = "right";
+	icon_state = "right";
```

Comment:
so i have to ask

why use /southleft and then edit the vars to be /southright?

Reply:
i dunno, i figured this was how you were supposed to do it since that's how i saw it handled everywhere else.